src/common/snippets/include/snippets/utils.hpp

```diff
@@ -29,7 +29,9 @@ ov::PartialShape get_port_planar_shape(const Output<Node>& out);
 ov::PartialShape get_reordered_planar_shape(const ov::PartialShape& shape, const std::vector<size_t>& layout);
 
 // Copy runtime info using default ngraph method but delete PortDescriptors which may be transferred after copying
-void safe_copy_runtime_info(const std::shared_ptr<ov::Node>&, const std::shared_ptr<ov::Node>& to);
+void safe_copy_runtime_info(const std::shared_ptr<ov::Node>& from, const std::shared_ptr<ov::Node>& to);
```

Comment:
Do I get it right that `safe` means `disregard PortDescriptors`?
So the idea here is here is that we copy everything except for `PortDescriptors`?

Reply:
Yeah, you're right. It was previous idea